rootfs/etc/profile.d/aws-vault.sh

```diff
@@ -51,7 +51,7 @@ if [ "${AWS_VAULT_ENABLED:-true}" == "true" ]; then
 				--reverse \
 				--prompt='-> ' \
 				--header 'Select AWS profile' \
-				--query "${AWS_ORG:-${NAMESPACE}}-" \
+				--query "${ASSUME_ROLE_INTERACTIVE_PREFIX:-${NAMESPACE}-${STAGE}-}" \
```

Comment:
```suggestion
				--query "${ASSUME_ROLE_QUERY:-${NAMESPACE}-${STAGE}-}" \
```

What about calling it `QUERY` since that's what `fzf` calls it?

Reply:
(it's also shorter)

Reply:
The ENV var needs to make sense in the global geodesic scope. I want to include `INTERACTIVE` because it only applies to the interactive role chooser